test/handler.test.ts

```diff
@@ -1,18 +1,12 @@
-import { handleRequest } from '../src/handler'
-import makeServiceWorkerEnv from 'service-worker-mock'
+import { someHandler } from "../src/some-handler";
 
-declare var global: any
+declare const global: typeof globalThis;
```

Comment:
global won't even be available without node-compat, so maybe just delete this? (I think it worked in the previous one because of webpack)

Reply:
Yeah it was being used by the service worker mock, adding a mocked worker to global scope. I removed that testing so this stopped making sense 😆